types/jest-expect-message/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for jest-expect-message 1.0
+// Project: https://github.com/mattphillips/jest-expect-message#readme
+// Definitions by: Mike Davydov <https://github.com/mike-d-davydov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
+
+/// <reference types="jest"/>
+
+export {};
+
+declare global {
```

Comment:
Can you try removing lines 9 and 11? By default if you have the remainder of the code, it will find the `jest` namespace in the global scope and augment it.

Reply:
Unfortunately, I can not, because it is exactly what I am trying to do: jest-expect message is a package that auguments jest `expect` by adding extra argument, it is the whole point. If you look at installation instructions for that package, they tell people to include jest-expect as Jest setup script. So if person uses that package, they are supposed to have an augumented "expect"

Reply:
Thanks! Looks like it works.